terraform/modules/aws-brig-email-sending/README.md

```diff
@@ -0,0 +1,18 @@
+Terraform module: Brig to enable email sending
+==============================================
+
+This module creates AWS resources (SES, SQS, SNS) enabling Brig to send emails.
+
```

Comment:
This module stores sensitive secrets inside terraform state file. Please add a comment in the readme warning people about this (i.e. ensure to encrypt the terraform state).

This module is also untested, and while we can merge an initial version to test the functionality and improve on subsequent PRs, please indicate that this code is not ready yet in the readme. See also the readmes of the other terraform modules for this.